pkgs/by-name/li/libvibrant/package.nix

```diff
@@ -0,0 +1,32 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, makeWrapper
+, libX11
+, libXrandr
+, linuxPackages
+}:
+
+stdenv.mkDerivation rec {
+  pname = "libvibrant";
+  version = "2100c09";
+
+  src = fetchFromGitHub {
+    owner = "libvibrant";
+    repo = pname;
+    rev = version;
+    hash = "sha256-nVODwP/PQgYBTHnSplgrkdNOLsF7N+vZ8iPL7gArVNY=";
+  };
+
+  buildInputs = [ libX11 libXrandr linuxPackages.nvidia_x11.settings.libXNVCtrl ];
```

Comment:
Is it limited to NVIDIA?

Reply:
No.
https://github.com/libvibrant/libvibrant/#compatibility
For nvidia cards it does require libXNVCtrl